doc/application_workflow/harbor_registry/harbor_with_fqdn.md

```diff
@@ -0,0 +1,223 @@
+.........................................................................++
+writing new private key to 'ca.key'
+-----
+You are about to be asked to enter information that will be incorporated
+into your certificate request.
+What you are about to enter is what is called a Distinguished Name or a DN.
+There are quite a few fields but you can leave some blank
+For some fields there will be a default value,
+If you enter '.', the field will be left blank.
+-----
+Country Name (2 letter code) [AU]:US
+State or Province Name (full name) [Some-State]:California
+Locality Name (eg, city) []:San Francisco
+Organization Name (eg, company) [Internet Widgits Pty Ltd]:VMware
+Organizational Unit Name (eg, section) []:vic
+Common Name (e.g. server FQDN or YOUR name) []:<Habor FQDN here!>
+Email Address []:
+loc@Devbox:~/mycerts$ ls
+ca.crt  ca.key
+```
+
+Generate a certificate signing request:  (FQDN is required. Sample data for demo purposes)
+
+```
+loc@Devbox:~/mycerts$ openssl req -newkey rsa:4096 -nodes -sha256 -keyout server.key -out server.csr
+Generating a 4096 bit RSA private key
+.....................................................................................................................................................................++
+.....................................................................++
+writing new private key to 'server.key'
+-----
+You are about to be asked to enter information that will be incorporated
+into your certificate request.
+What you are about to enter is what is called a Distinguished Name or a DN.
+There are quite a few fields but you can leave some blank
+For some fields there will be a default value,
+If you enter '.', the field will be left blank.
+-----
+Country Name (2 letter code) [AU]:US
+State or Province Name (full name) [Some-State]:California
+Locality Name (eg, city) []:San Francisco
+Organization Name (eg, company) [Internet Widgits Pty Ltd]:VMware
+Organizational Unit Name (eg, section) []:vic
+Common Name (e.g. server FQDN or YOUR name) []:<Habor FQDN here!>
+Email Address []:
+
+Please enter the following 'extra' attributes
+to be sent with your certificate request
+A challenge password []:password
+An optional company name []:
+loc@Devbox:~/mycerts$ ls
+ca.crt  ca.key  server.csr  server.key
+```
+Notice above, we used the name server.key and server.csr to make it clear.  This is a deviation from the Harbor documentation.
+
+Sign the server certificate:  (subjectAltName is required.  It is the IP from above.)
+
+```
+loc@Devbox:~/mycerts$ mkdir demoCA
+loc@Devbox:~/mycerts$ cd demoCA
+loc@Devbox:~/mycerts/demoCA$ touch index.txt
+loc@Devbox:~/mycerts/demoCA$ echo '01' > serial
+loc@Devbox:~/mycerts/demoCA$ cd ..
+loc@Devbox:~/mycerts$ echo subjectAltName = IP:HarborVM_IP > extfile.cnf
+
+loc@Devbox:~/mycerts$ openssl ca -in server.csr -out server.crt -cert ca.crt -keyfile ca.key -extfile extfile.cnf -outdir .
+Using configuration from /usr/lib/ssl/openssl.cnf
```

Comment:
Ahh, I think last time I saw it it wasn't wrapped to a newline so I thought it may have been a rogue paste. 👍